file_formats/fip.md

```diff
@@ -14,77 +14,98 @@ Following SciComp standards, FIP data should be saved in their own folder named
 
 ### File format 
 
-In most cases, FIP fiber photometry data will be stored in 3 photometry readout CSV files (extracted traces + timestamp; 6 time-series), 3 bin files (raw camera-detector movie file), and 2 ROI coordinate CSVs.
-
-For example:
+In most cases, FIP data will be saved in `CSV` files, where each file corresponds to a different channel in the photometry rig. In addition to the timeseries fluorescence data, files containing metadata and raw image data are also available. A single session of FIP data should be organized under a the fib directory. There must be sub-directories prepended with `fip_` for a session recording. There may be multiple `fip_` directories to handle cases where a FIP session crashes and needs to re-start recording. Therefore, a FIP session directory should look like this:
```

Comment:
Should `fip_<blah>` just be a datetime?

Reply:
@arielleleon raised the same point during the last round up.
My feeling is that, as file format to be parsed, I don't see the advantage of enforcing this. At least as something that can be referenced my users outside the Allen to save their own data. 

This metadata (along with details of the rig and acquisition) information should belong to the aind-data-schemas not the file-format and not duplicated via file name metadata, imo.

Alternatively, if you want to move in this direction I would rather just enforce this on every single folder (BIDS style) as part of the core standard. Otherwise I think we will just end up with a cacophony of non consistent file naming patterns. 

Anyway it's an easy addition for pipelines in the Allen, I would just not enforce/rely on it for processing.

Reply:
agree - this is just for unique naming. I mostly don't want to encouraging emedding secret semantics in this folder names.